### ISSUE TITLE
Fix missing parameter in client API docs

### ISSUE THREAD

**Issue by jeromekelleher** [ga4gh/server#674]
The Python client API is a simple interface to the GA4GH server. The API is documented [here](http://ga4gh-reference-implementation.readthedocs.org/en/develop/api.html), and this documentation is largely derived from the comments in the Avro source. 

Copying the comments for the parameter definitions automatically was not possible, and so there are a number of gaps that need to be filled. Using the comments for the official documention at 

https://github.com/ga4gh/schemas/tree/master/src/main/resources/avro

fill these in.

- Start by opening ``ga4gh/client.py``, and find the ``searchReferences`` function. Using the official documentation in [referencemethods.avdl](https://github.com/ga4gh/schemas/blob/master/src/main/resources/avro/referencemethods.avdl) fill in the TODOs as follows
```python
    def searchReferences(
            self, referenceSetId, accession=None, md5checksum=None):
        """
        Returns an iterator over the References fulfilling the specified
        conditions from the specified Dataset.
        
        :param str referenceSetId: The ReferenceSet to search.
        :param str accession: If not None, return the references for which the 
            `accession` matches this string (case-sensitive, exact match).
        :param str md5checksum: If not None, return the references for which the    
             `md5checksum` matches this string (case-sensitive, exact match).
        :return: An iterator over the :class:`ga4gh.protocol.Reference`
            objects defined by the query parameters.
        """
 ```

- Build the documentation by cd'ing to the ``docs`` directory, and running ``make``. Check the output of the build by opening ``docs/_build/html/api.html`` in your web browser. Ensure that there aren't any warnings from Sphinx as these will be treated as errors by out Travis CI script.

- For all the other TODO documentation items, find the corresponding documentation in the schemas and fill in the parameter documentation appropriately.

**Comment by sguthrie** [comment]
Working on this!

**Comment by sguthrie** [comment]
@jeromekelleher, would you prefer a direct copy-paste between the schemas documentation or would you like some edits to make the new documentation fit stylistically?

**Comment by dcolligan** [comment]
Fixed in #730
